app/views/articles/show.html.erb

```diff
@@ -155,11 +155,12 @@
                 <% if @article.published_timestamp.present? %>
                   <%= local_date(@article.published_timestamp, show_year: @article.displayable_published_at.year != Time.current.year) %>
                 <% end %>
-                <% if @second_user.present? %>
-                  <em>with <b><a href="<%= @second_user.path %>"><%= @second_user.name %></a></b></em>
-                <% end %>
-                <% if @third_user.present? %>
-                  <em> and <b><a href="<%= @third_user.path %>"><%= @third_user.name %></a></b></em>
+                <% if @article.co_author_ids.present? %>
+                  <% if @article.co_author_ids.count == 1 %>
+                    <em>with <b><a href="<%= @article.co_authors.first.path %>"><%= @article.co_authors.first.name %></a></b></em>
+                  <% elsif @article.co_author_ids.count > 1 %>
```

Comment:
```suggestion
                  <% else %>
```
Since you already have checked if `@article.co_author_ids.present?`, and have an `if` for when the `count` is exactly `1`, you can just do a plain `else` here, because you can be sure that it will be greater than `1` but also not `0`.

Reply:
Good call!